linux/Dockerfile

```diff
@@ -66,6 +67,8 @@ RUN apt update -qy && apt install -y \
     pkg-config libssl-dev libcurl4-openssl-dev libexpat-dev libpq-dev libz-dev \
     rpm tar gettext autopoint autoconf clang libtool-bin \
     pkg-config flex meson selinux-basics squashfs-tools gpg xz-utils gnupg2 patchelf cpio \
+    # xsltproc is required by libvirt-sdk used in the micro-vms scenario
+    xsltproc \
```

Comment:
This list of dependencies is convoluted. I see duplicates: `pkg-config`. I know it was like this before, but what about taking the opportunity to sort it, one entry per line?